lib/views/activites/photograph_activity_view.dart

```diff
@@ -111,7 +111,7 @@ class _PhotographActivityViewState extends State<PhotographActivityView> {
                       padding: const EdgeInsets.only(top: 20.0),
                       child: Container(
                         height:
-                            (MediaQuery.of(context).size.height / 100) * 45.2,
+                            Screen.height(context, percentage: 45.2),
```

Comment:
Can we make these round numbers. 45.2% is far to specific. Why not 45%?

Reply:
When I converted them to be based off of width or height originally I tired to get them as close as possible to the original value it wont make much of a difference of small devices but on larger devices with larger widths it does make a difference so I think we should keep them unrounded maybe to 1 dp if we need to but I am happy with them as they are